### ISSUE TITLE
Update compliance date for accessibility

### ISSUE THREAD

**Issue by AJSMITH1512** [nhsuk/nhsuk-service-manual#959]
### What

The accessibility pages have a refence to the date for compliance and this needs to be updated to reflect the passing of time.

### Specific Issue

**Product and Delivery**

Recommended Update:

**Publish an accessibility statement**

New websites must have an accessibility statement from September 2019. Existing websites need one by September 2020, in line with new accessibility regulations.

Revise - suggest all websites now need to have an accessibility statement in line with the 2018 accessibility regulations.

### To do

- [x] Draft updated content
- [x] Agree wording and update page
- [ ] Double-check content (2i by a content designer)
- [ ] Release update

**Comment by sarawilcox** [comment]
Should be reworded as follows: 

> All websites must have an accessibility statement, in line with accessibility regulations. Read more about [the accessibility requirements for public sector bodies on GOV.UK](https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps).

Hi @Tosin-Balogun, if you'd like to make this change, you'll need to do the following: 
- amend this partial: /accessibility/partials/publish-an-accessibility-statement.njk
- update the date at the bottom of this page: /accessibility/product-and-delivery
- update the What's new page and the what's new/updates page

See David's March updates on this page: https://nhsuk-service-manual-dev-uks.azurewebsites.net/whats-new

We need to add a line:
Accessibility guidance | Updated guidance on accessibility statements (with a link)

It's best if you can do this later this week so it doesn't clash with releasing the review date pattern today or tomorrow. I can check your PR when I'm back at work next week and then we'll need another content designer to 2i it before release.

Thanks a lot.

**Comment by Tosin-Balogun** [comment]
@sarawilcox - I have opened a branch for this https://github.com/nhsuk/nhsuk-service-manual/tree/Update-compliance-date-for-accessibility_%23959

I have started to action the above in the new branch, will probably make the pull request post release of the date pattern

**Comment by chrimesdev** [comment]
This is done
